LGTM "could not build merge commit"

sarif-sdk builds fine from our command line, and also from our ADO build pipeline. But in PR #2054, the LGTM analysis started to fail  after a [commit](4ae6aeb0a3689a6b9799b640aedb7446c456f7a1) where I removed certain NuGet references from a project which, due to the changes in this PR, should no longer have needed them. Restoring the NuGet references, in [this commit](https://github.com/microsoft/sarif-sdk/pull/2054/commits/8f631e607576da9ede325c007b62a5da966e61da), did not help.
 
The failure is that it "could not build the merged commit" with errors like this:
 
[2020-09-02 00:11:38] [build] /opt/src/src/Shared/CommonAssemblyInfo.cs(12,12): error CS0246: The type or namespace name 'ComVisibleAttribute' could not be found (are you missing a using directive or an assembly reference?) [/opt/src/src/Sarif.Multitool.Library/Sarif.Multitool.Library.csproj]

@michaelcfanning FYI

--- a/src/Sarif.Multitool.Library/Sarif.Multitool.Library.csproj
+++ b/src/Sarif.Multitool.Library/Sarif.Multitool.Library.csproj
@@ -13,6 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="Microsoft.Json.Pointer" Version="1.1.2" />
   </ItemGroup>
 

--- a/src/Sarif.Sdk.CodeQL.sln
+++ b/src/Sarif.Sdk.CodeQL.sln
@@ -17,6 +17,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sarif.WorkItems", "Sarif.Wo
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WorkItems", "WorkItems\WorkItems.csproj", "{175F37F3-E94B-4B4F-AA46-F0BF272CBD74}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sarif.Multitool.Library", "Sarif.Multitool.Library\Sarif.Multitool.Library.csproj", "{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
@@ -99,6 +101,18 @@ Global
 		{175F37F3-E94B-4B4F-AA46-F0BF272CBD74}.Release|x64.Build.0 = Release|Any CPU
 		{175F37F3-E94B-4B4F-AA46-F0BF272CBD74}.Release|x86.ActiveCfg = Release|Any CPU
 		{175F37F3-E94B-4B4F-AA46-F0BF272CBD74}.Release|x86.Build.0 = Release|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Debug|x64.Build.0 = Debug|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Debug|x86.Build.0 = Debug|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Release|Any CPU.Build.0 = Release|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Release|x64.ActiveCfg = Release|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Release|x64.Build.0 = Release|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Release|x86.ActiveCfg = Release|Any CPU
+		{4A60ED16-36B4-435E-88A4-36B39CC6DFD7}.Release|x86.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE

